prime-router/metadata/schemas/WATERS/waters-covid-19.schema

```diff
@@ -280,7 +280,7 @@ elements:
   # Time that waters recieved the record
   - name: waters_receive_date
     type: DATETIME
-    csvFields: [{ name: TXNTIMESTAMP}]
+    csvFields: [{ name: TXNTIMESTAMP, format: yyyyMMddhhmmss}]
```

Comment:
This actually didn't fix the problem Oracle team was having.  I created a ticket to fix our DATETIME formatting.